.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
please could you remove this .idea/ files or am I missing something? I think we should not add these files, our main .gitignore is the one at the root of our folder

Reply:
Remove and resolved